README.md

```
@@ -213,33 +257,73 @@
+사용자가 실제로 보는 화면과 행동을 기준으로 테스트한다.
+
+- 카드 목록을 조회하는 경우, 요청하는 동안 스켈레톤 UI를 보여준다.
+- 카드 목록 조회 성공 시, 카드 목록이 존재하면 등록된 카드 목록을 표시한다.
+- 카드 목록 조회 성공 시, 카드 목록이 비어 있으면 빈 상태 UI와 카드 등록 버튼을 표시한다.
+- 카드 목록 조회 실패 시 에러 UI와 다시 시도 버튼을 표시한다.
+- 카드 삭제 요청이 성공하면 카드가 목록에서 사라진다.
+- 카드 등록 실패 시 서버 에러 메시지를 해당 필드 또는 form-level 영역에 표시한다.
 
 ## 폴더 구조
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)